opencti-platform/opencti-graphql/src/database/grakn.js

```diff
@@ -271,7 +284,6 @@ export const queryAttributeValues = async type => {
     return buildPagination(5000, 0, result, 5000);
   } catch (err) {
     logger.error('[GRAKN] queryAttributeValues error > ', err);
-    await closeReadTx(rTx);
```

Comment:
why removing this ?

Reply:
@richard-julien It seems that if there is an error during a Grakn query, the transaction is closed automatically. By doing this, the await never end because the closeReadTx did not respond.

Reply:
Ok